src/solver/simulation/sim_calcul_economique.cpp

```diff
@@ -318,17 +318,29 @@ void SIM_InitialisationProblemeHebdo(Data::Study& study,
     problem.LeProblemeADejaEteInstancie = false;
 }
 
-void preparerBindingConstraint(const PROBLEME_HEBDO &problem, uint numSpace, int PasDeTempsDebut,
-                               const BindingConstraintsRepository &bindingConstraints, const uint weekFirstDay, int pasDeTemps) 
+void preparerBindingConstraint(const PROBLEME_HEBDO &problem, int PasDeTempsDebut,
+                               const BindingConstraintsRepository &bindingConstraints,
+                               const BindingConstraintGroupRepository &bcgroup,
```

Comment:
A repository contains several element.
bcgroup would be better name either as a plurial or with "repository" in its name
E.g. bcGroups, bcGroupsRepository, bindingConstrainsGroups, bindingConstraintsGroupsRepository